content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md

```diff
@@ -22,35 +42,35 @@ Creating a vegetation layer is the first and most basic step in creating your dy
 
     The **Vegetation Layer Spawner** component is the core component that initializes the engine that spawns vegetation.
 
-1. Click **Add Required Component** and choose **Vegetation Reference Shape**.
+1. Click **Add Required Component** and choose **Shape Reference**.
 
-    ![Choose the Vegetation Reference Shape for your Vegetation Layer Spawner.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-add-shape.png)
+    ![Choose the Shape Reference for your Vegetation Layer Spawner.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-add-shape.png)
 
-    The **Vegetation Reference Shape** has no shape on its own. You must next create a child entity and add a **Shape component**, which you will reference in the **Vegetation Reference Shape**.
+    The **Shape Reference** has no shape on its own. You must next create a child entity and add a **Shape component**, which you will reference in the **Shape Reference**.
 
-1. Right-click **BasicCoverage**, select **Create Child Entity**, and name it **TestBox**.
+1. Right-click **BasicCoverage**, select **Create Entity**, and name it *TestBox*.
 
-1. Select **TextBox**, click **Add Component**, and select the **Box Shape** component.
+1. Select *TestBox*, click **Add Component**, and select the **Box Shape** component.
 
 1. Adjust the size and position of the shape so that it's large enough for your purposes and intersects with the ground.
 
     ![Adjust your shape to cover a sufficient area and intersect with the ground.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-adjust-shape.png)
 
-1. Select **BasicCoverage** and, in the **Vegetation Reference Shape** component, click the target symbol and select the **TestBox** entity.
+1. Select *BasicCoverage* and, in the **Shape Reference** component, click the target symbol and select the *TestBox* entity.
```

Comment:
We can add inline icons for interface elements.

```suggestion
1. Select *BasicCoverage* and, in the **Shape Reference** component, click the {{< icon "picker.svg" >}} **entity picker** icon and select the *TestBox* entity.
```

Reply:
Sweet, updated!